From cd067ab27f08f8744d1ce75225d56f8ac9631ab7 Mon Sep 17 00:00:00 2001
From: Rico Ganahl <rico.ganahl@bytesatwork.ch>
Date: Thu, 17 Aug 2023 14:27:45 +0200
Subject: [PATCH] configs: bytedevkit: enable gpio support

Used for GPIO test in production.
Not muxed AM62x GPIOs only work in input mode on AM62x and
can only be read by the gpio cmd.

Signed-off-by: Rico Ganahl <rico.ganahl@bytesatwork.ch>
---
 configs/am62x_bytedevkit_a53_defconfig | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/configs/am62x_bytedevkit_a53_defconfig b/configs/am62x_bytedevkit_a53_defconfig
index 33f18c7725..a2169dc0ac 100644
--- a/configs/am62x_bytedevkit_a53_defconfig
+++ b/configs/am62x_bytedevkit_a53_defconfig
@@ -12,6 +12,7 @@ CONFIG_BAW_COMMON_CMD_OPTIONS=y
 CONFIG_CMD_BAW_CONFIG=y
 CONFIG_ENV_SIZE=0x20000
 CONFIG_ENV_OFFSET=0xC0000
+CONFIG_DM_GPIO=y
 CONFIG_SPL_TEXT_BASE=0x80080000
 CONFIG_SPL_MMC_SUPPORT=y
 CONFIG_SPL_SERIAL_SUPPORT=y
@@ -57,7 +58,6 @@ CONFIG_CMD_ABOOTIMG=y
 CONFIG_CMD_MEMINFO=y
 CONFIG_CMD_BCB=y
 CONFIG_CMD_DM=y
-# CONFIG_CMD_GPIO is not set
 CONFIG_CMD_AB_SELECT=y
 CONFIG_CMD_PXE=y
 CONFIG_CMD_SYSBOOT=y
@@ -97,6 +97,7 @@ CONFIG_FASTBOOT_FLASH=y
 CONFIG_FASTBOOT_FLASH_MMC_DEV=0
 CONFIG_FASTBOOT_CMD_OEM_FORMAT=y
 CONFIG_TI_SCI_PROTOCOL=y
+CONFIG_DA8XX_GPIO=y
 CONFIG_SYS_I2C_OMAP24XX=y
 CONFIG_DM_MAILBOX=y
 CONFIG_K3_SEC_PROXY=y
-- 
2.39.2

